File: 프로그래머스/python/LV1/이상한 문자 만들기.py
def solution(s):
    answer = ""
    arr = s.split(" ")
    for i in range(len(arr)):
        temp = list(arr[i])
        for j in range(len(temp)):
            if j % 2 == 0:
                temp[j] = temp[j].upper()
            else:
                temp[j] = temp[j].lower()
        arr[i] = "".join(temp)

    return " ".join(arr)

File: 프로그래머스/javascript/LV1/이상한 문자 만들기.js
function solution(s) {
  let answer = [];
  let arr = s.split(" ");
  for (word of arr) {
    let newWord = "";
    for (let i = 0; i < word.length; i++) {
      if (i % 2 == 0) {
        newWord += word[i].toUpperCase();
      } else {
        newWord += word[i].toLowerCase();
      }
    }
    answer.push(newWord);
  }
  return answer.join(" ");
}

// 문자.toLowerCase() , 문자.toUpperCase() => 문자 대소문자 변경.
